card/lib/generators/card/USAGE

```diff
@@ -0,0 +1,21 @@
+Description
+
+Examples
+  wagn generate card new cards pre_fix
```

Comment:
card, new, cards, and pre_fix are all variables?  Can you choose some more plausible variable names?